tests/codegen/debuginfo-inline-callsite-location.rs

```diff
@@ -20,7 +20,7 @@
 #![crate_type = "lib"]
 
 #[no_mangle]
-extern "C" fn add_numbers(x: &Option<i32>, y: &Option<i32>) -> i32 {
+extern "C-unwind" fn add_numbers(x: &Option<i32>, y: &Option<i32>) -> i32 {
```

Comment:
This seems like a bit of an arbitrary change. Is it needed for this test to pass?

Reply:
Without this change there'll be terminate guards generated, so the `CHECK: tail call` above wouldn't match.

Reply:
A lot of other tests seem to have similar problems and you added `-C panic=abort` there. Why use a different approach here? Or is it a similar problem?

Reply:
There's no particular reason. The other tests have a lot more extern functions so I used `-C panic=abort` instead of changing all of them to `C-unwind`. I have now fixed this test to also use `-C panic=abort`.